DataFormats/Parameters/include/DataFormatsParameters/LHCIFData.h

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file LHCIFData.h
+/// \brief container for the LHC InterFace data
+
+#ifndef O2_GRP_LHCIFDATA_H
+#define O2_GRP_LHCIFDATA_H
+
+#include <Rtypes.h>
+#include <string>
+#include <cstdint>
+
+namespace o2
+{
+namespace parameters
+{
+
+class LHCIFData
+{
+ public:
+  LHCIFData() = default;
+  ~LHCIFData() = default;
+
+  std::pair<float, int32_t> getBeamEnergy() const { return mBeamEnergy; }
```

Comment:
why the time is float?

Reply:
This is provided like this from LHC_IF. I asked Antonio, and he said that this is what they get. Should I change it to uint64_t, as all our timestamps?

Reply:
yes, we better stick to the same types and units: long, microseconds (but the CCDB validity range in ms).

Reply:
done